data/changes/bantu.txt

```diff
@@ -15,26 +15,26 @@ Bantu
 - [*p̱ *t *k] / [{p s̪} t̪] / _ [*i̧ *u̧]
 - *ʃ / s̪ / _ *i̧
 + Inferred from modern reflexes
-- [*ḇ *ḏ *g̱] / {*ḇ z̪} {*ḏ z̪} {*g̱ z̪} / _ [*i̧ *u̧]
```

Comment:
This is something I’ve been unsure about. So far I’ve been transcribing *CC clusters in sources as `*C *C`, on the basis that the asterisk implies both segments are reconstructed. But I suppose you could make a reasonable case for it being `*ḇ z̪` if the second protophoneme is definitely the same as the modern one.

Reply:
We wouldn't know if it was _really_ the same, maybe then it was a genuine [z̪] rather than [ð]. But this isn't indicated anywhere, so in doubt I'd rather have it be the modern phoneme.

Reply:
Hmm, that’s fair. We can just keep it as the modern phoneme then.

Reply:
Actually, on second thoughts… a few lines below this is `[{*ḇ z̪} {*ḏ z̪} {*g̱ z̪}] / z̪`. So I think I’d prefer to keep this as reconstructed *z̪ after all, on the basis that we can’t be completely confident about that intermediate stage.